community/overseerr/1.32.5/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
I'm assuming everything in the community/* folder should not be included in the PR as it's generated from the `create_app.sh` script and is probably built during a CI process somewhere?